models/prices/prices_tokens.sql

```diff
@@ -273,47 +273,7 @@ VALUES
     ("wom-wombat-exchange", "bnb", "WOM", "0xad6742a35fb341a9cc6ad674738dd8da98b94fb1", 18),
     ("hay-hay", "bnb", "HAY", "0x0782b6d8c4551b9760e74c0545a9bcd90bdc41e5", 18),
     ("bnbx-stader-bnbx", "bnb", "BNBx", "0x1bdd3cf7f79cfb8edbb955f20ad99211551ba275", 18),
-    ("air-aircoin", "bnb" ,"AIR", "0xd8a2ae43fd061d24acd538e3866ffc2c05151b53", 18),
```

Comment:
this seems unintentional -- why did the latest commit remove all these prices?

Reply:
My bad . I have no idea how that happened

Reply:
Let me fix